operations/tempo-mixin/config.libsonnet

```diff
@@ -25,6 +25,9 @@
       p99_request_exclude_regex: 'metrics|/frontend.Frontend/Process|debug_pprof',
       outstanding_blocks_warning: 100,
       outstanding_blocks_critical: 250,
+      // Partition lag thresholds in seconds
```

Comment:
Can we add a setting to filter the groups?  For the case that there are other consumer groups running components in the same namespace, but unrelated.  i.e. Maybe a regex like `partition_lag_group_filter: "(metrics-generator|block-builder)"`

Reply:
Done!